File: src/main/java/com/example/today_i_baekjoon/domain/Holiday.java
package com.example.today_i_baekjoon.domain;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDate;

@Getter
@Builder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@Entity
@Table(name = "holidays")
public class Holiday {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false, unique = true)
    private LocalDate date;
}

File: back/src/main/kotlin/com/gidskql6671/today_i_baekjoon/domain/Holiday.kt
package com.gidskql6671.today_i_baekjoon.domain

import jakarta.persistence.*
import java.time.LocalDate

@Entity
@Table(name = "holidays")
class Holiday(@Column(nullable = false, unique = true) var date: LocalDate) {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null
}